src/components/Navbar.tsx

```diff
@@ -0,0 +1,105 @@
+import { FaUserCircle } from "react-icons/fa";
+import { useDispatch } from "react-redux";
+import { useNavigate } from "react-router-dom";
+import { useState } from "react";
+import { clearAuth } from "../modules/auth/authSlice";
+import AuthLogo from "./AuthLogo";
+import ThemeToggle from "../modules/theme/ThemeToggle";
+
+const Navbar = () => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [isDropdownOpen, setIsDropdownOpen] = useState(false);
+
+  // Handle logout
+  const handleLogout = () => {
+    dispatch(clearAuth());
+    localStorage.removeItem("accessToken");
```

Comment:
We can call `localStorage.clear()` which will clear all the keys in the local storage.

Reply:
Thanks for this new info, didnt know it existed!